twarep/electrifly
# Auto refresh data

## Checklist
**Note:** Put an x into the box like this [x] to check these off

## What I Did
What files git you edit and why/for what purpose?
- modified scraper.py to write to a file everytime it is run
- modified crontab in linux sever under peter's user to run the scraper.py script hourly
- modified app.py to output the last time the scraper ran

## How I Did it?
What packages did I install to accomplish this goal? What libraries did I need that the team might not know?
Write anything in this section that is technically related that everyone on the team might not know or is not
trivial.
- crontab, python

## How to Test?
Manual testing? Automated testing? E2E testing? Smoke tests?
- run scraper.py and check if the date/time is updated on the upload screen on app.py 
- -> at the bottom "Last Data Retrieval" 
<img width="643" alt="Screenshot 2023-11-23 at 12 32 07 PM" src="https://github.com/twarep/electrifly/assets/71115362/9c1395a1-8bf1-4501-b59f-e924a6d869ee">

- login to peter's user on the linux server and check if it's running the scraper.py every hour
<img width="820" alt="Screenshot 2023-11-23 at 12 34 01 PM" src="https://github.com/twarep/electrifly/assets/71115362/6629d203-6a8d-4725-b5b4-df2a9c6a3694">



### Relevant Links
Any tutorials or other resources used to help with this task. You can also cite the documentation.
